src/mocks/qaSection.mock.json

```diff
@@ -75,10 +75,39 @@
         {
           "type": "paragraph--wysiwyg",
           "id": "f6dfd8da-877f-46bd-9707-28861c27003e",
-          "resourceIdObjMeta": {
-            "target_revision_id": 3011,
-            "drupal_internal__target_id": 2754
-          }
+          "drupal_internal__id": 2754,
+          "drupal_internal__revision_id": 3011,
```

Comment:
When I started using the specific paragraph type in `ParagraphQaSection`, these unhydrated mock objects would no longer cut it for keeping TypeScript happy.